video/mp_image.c

```diff
@@ -525,6 +533,8 @@ void mp_image_copy_attributes(struct mp_image *dst, struct mp_image *src)
                 memcpy(dst->planes[1], src->planes[1], MP_PALETTE_SIZE);
         }
     }
+    if (src->icc_profile)
+        dst->icc_profile = av_buffer_ref(src->icc_profile);
```

Comment:
I'm not sure - what errors are we supposed to handle? Wouldn't the function just return NULL if it fails, which is the same as “no ICC profile”? I don't know what else we could be doing.